pkg/scheduler/api/resource_info.go

```diff
@@ -77,9 +76,7 @@ func NewResource(rl v1.ResourceList) *Resource {
 			r.MaxTaskNum += int(rQuant.Value())
 		default:
 			//NOTE: When converting this back to k8s resource, we need record the format as well as / 1000
-			if v1helper.IsScalarResourceName(rName) {
-				r.AddScalar(rName, float64(rQuant.MilliValue()))
-			}
+			r.AddScalar(rName, float64(rQuant.MilliValue()))
```

Comment:
Why remove the validation?

Reply:
> Why remove the validation?



> Why remove the validation?

This may be the easiest way to support  `ephemeral-storage`

Reply:
> Why remove the validation?



> Why remove the validation?

This may be the easiest way to support  `ephemeral-storage`

Reply:
> > Why remove the validation?
> 
> This may be the easiest way to support `ephemeral-storage`

That may be a little bit rough. Besides, Considering `ephemeral-storage` as part of scalar resource is not so reasonable.